helpers/lib/fonts.js

```diff
@@ -84,58 +86,77 @@ const fontProviders = {
  * @returns {Object.<string, Array>|string}
  */
 module.exports = function(format, themeSettings, handlebars, options) {
-    
+
     const collectedFonts = {};
-    _.each(themeSettings, function(value, key) {
-        //check that -font is on end of string but not start of string
+    const themeSettingsKeys = Object.keys(themeSettings);
```

Comment:
This portion of the change has become really complex, and I'm pretty sure something is wrong with this implementation. Perhaps we should strip this part out and save it for another PR?